protocol/agent-protocol.md

```diff
@@ -78,13 +79,17 @@ The procedure of establishing a duplex connection is explained on the example of
 4. Bob sends `JOIN` command with the invitation as a parameter to agent B to accept the connection.
 5. Establishing Alice's SMP queue (with SMP protocol commands):
   - Agent B sends unauthenticated message to SMP queue with ephemeral key that will be used to authenticate commands to the queue, as described in SMP protocol.
-  - Agent A receives the KEY and secures the queue.
+  - Agent A receives the KEY and Bob's info.
```

Comment:
capital KEY may be confused with SMP's KEY (secure) command - maybe replace with "... receives the SMP confirmation containing the key and Bob's info"